blockmanager.go

```diff
@@ -534,7 +573,7 @@ func (b *blockManager) current() bool {
 func (b *blockManager) handleBlockMsg(bmsg *blockMsg) {
 	// If we didn't ask for this block then the peer is misbehaving.
 	blockSha := bmsg.block.Sha()
-	if _, ok := bmsg.peer.requestedBlocks[*blockSha]; !ok {
+	if _, exists := bmsg.peer.requestedBlocks[*blockSha]; !exists {
```

Comment:
Please rename the field in the message if it's peer info.  This is confusing to read since it looks like a `*peer.Peer` instead of `*peerInfo`.